service/app/mapper/RequestMapper.java

```diff
@@ -25,11 +25,17 @@
    */
   public static <T> Object mapRequest(JsonNode requestData, Class<T> obj) throws RuntimeException {
 
+    if(requestData==null)
+		throw new ProjectCommonException(
```

Comment:
Write a public static method to create clientException . Method will take ResponseCode enum as an argument and return ProjectCommonException.

Reply:
Commited changes in this branch and created a  method in ProjectUtil  class Sunbird-utils project.